object_database/web/cells/cells.py

```diff
@@ -512,6 +513,13 @@ def _recalculateCells(self):
                     if not node.garbageCollected:
                         self._recalculateSingleCell(node)
 
+        # Any Subscribeds that are set to broadcast
+        # and that have been updated should also
+        # make sure their parent Cell is updated.
+        for node in self._nodesToBroadcast:
+            if isinstance(node, Subscribed) and node.wasUpdated:
+                node.parent.wasUpdated = True
```

Comment:
what happens if you have a Subscribed in a Subscribed in a Subscribed? I think you probably need to walk up the tree until you see a node that's not a subscribed, marking wasUpdated the whole way.

Reply:
Good catch. Updated in the latest push.